src/debug/telemetry/telemetry2_slot.h

```diff
@@ -54,6 +54,9 @@
 
 #include <stdint.h>
 
+// HACK: This should be in zephyr/soc/intel/intel_adsp/common/include/adsp_debug_window.h
```

Comment:
So if the payload is application specific we should be in SOF header, but RTOS specific in Zephyr header - if it s both then we cant keep in in application header.